samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <logging/log.h>
```

Comment:
From @pfalcon in #26719:
big_http_download is a POSIX-compatible sample (i.e. written with purely POSIX API, and portable to any POSIX system). Please run make -f Makefile.posix in the sample dir to ensure it remains so. (This change likely breaks that.)

Reply:
@pfalcon I've tested this example for POSIX on latest master and it is not compiles:

Compile command: `make -f Makefile.posix`

Error:

`cc src/big_http_download.c -lmbedcrypto -o big_http_download
src/big_http_download.c:12:10: fatal error: 'mbedtls/md.h' file not found
#include "mbedtls/md.h"
         ^~~~~~~~~~~~~~
1 error generated.`

Thus currently not my approach breaks the code!

Reply:
To avoid not supported logging on POSIX following will be done:

`#if defined(__ZEPHYR__) && !defined(CONFIG_POSIX_API)

#include <logging/log.h>
#define LOG_LEVEL LOG_LEVEL_DBG
LOG_MODULE_REGISTER(big_http_download);

#endif`